frontend/src/components/common/RenderGroupedSelect.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Form, Select } from "antd";
+import CustomPropTypes from "@/customPropTypes";
+
+/**
+ * @constant RenderSelect - Ant Design `Select` component for redux-form - used for small data sets that (< 100);
+ * There is a bug when the data sets are large enough to cause the dropdown to scroll, and the field is in a modal.
+ * In the case where the modal cannot scroll, it is better to pass in the prop doNotPinDropdown.  It allows the
+ * dropdown to render properly
```

Comment:
This doesn't look like the right comment

Reply:
Changed a bit to reflect the grouped aspect of the select.